Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/GDSInformation/GDSInformation.xib

```diff
@@ -107,9 +107,6 @@
                             <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
                             <inset key="imageEdgeInsets" minX="0.0" minY="0.0" maxX="2.2250738585072014e-308" maxY="0.0"/>
                             <state key="normal" title="Confirm your identity another way"/>
-                            <userDefinedRuntimeAttributes>
```

Comment:
@mcgarj we can see here it was set to a `userDefinedRuntimeAttributes` hence why the view model wasn't doing anything as this would of been called after! (appreciate you didn't add this, just more from a debugging perspective for future :) )

Reply:
Now I know for next time where to look if everywhere else looks okay!